### ISSUE TITLE
The scribble documentation doesn't say how to install scribble

### ISSUE THREAD

**Issue by llimllib** [racket/scribble#342]
The title is the whole issue, I found the documentation for scribble and I'm interested, I even have racket installed already, but I have no idea how to install scribble and the documentation assumes you have a `scribble` binary already.

**Comment by samth** [comment]
If you've installed Racket there should be a parallel scribble binary. There is also a raco scribble command that should work as well if you can run raco.

**Comment by llimllib** [comment]
- I installed racket via homebrew, and there isn't a scribble binary
- `raco pkg install scribble' worked, though it took 11 minutes (!) and prompted me to accept package installs 34 (!!) different times ([full log](https://gist.github.com/fef56610fb2dda8510b26e3074aaed8c))

Anyway, I think putting "it's either in your distribution or do `raco pkg install scribble`" into the docs would maybe be a good idea - but it's just a suggestion and if you don't want to no worries, close away

**Comment by jryans** [comment]
The default `racket` package on Homebrew is only a "minimal" distribution (and has even been renamed to `minimal-racket` to make it a bit more obvious), so it's missing many of the extra packages and commands that usually come with Racket, including `scribble`.

There's a Homebrew cask package which does include a full distribution. You could do the following if you wanted to switch over:

1. `brew remove racket`
2. `brew install --cask racket`

**Comment by llimllib** [comment]
It's cool, I have it installed now! Just think it might be handy to include installation instructions in the docs for the tool

**Comment by rfindler** [comment]
@llimllib We have a bit of a quandry here-- the easiest/simplest way to install on a mac (namely following the download link on the racket-lang.org site and then dragging to the Applications folder) does not require any steps. Also, there are many many many pages that hold the same status as the scribble documentation, in the sense that someone coming to them who has somehow used one of the more complex ways to install racket would need to take additional steps. So it doesn't seem like the right move to just change the scribble documentation. 

Maybe the right fix is to make "brew install racket" actually have scribble in it (but perhaps, from @jryans comment, this has been done already).

**Comment by jasonhemann** [comment]
@rfindler, the homebrew messages about the deprecated formula `racket` are IMO a little opaque (cf https://github.com/orgs/Homebrew/discussions/3510), and so issues like this are a downstream consequence.

**Comment by rfindler** [comment]
I'm not familiar enough with brew to understand what that issue is talking about, @jasonhemann . Can you explain why "brew install racket" should install something derived from minimal-racket instead of racket? Or are you saying something else?

**Comment by samth** [comment]
The problem is that `brew install racket` prints a deprecation warning and tells people to install `minimal-racket` instead (that's what's described in the issue).

**Comment by jasonhemann** [comment]
> The problem is that `brew install racket` prints a deprecation warning and tells people to install `minimal-racket` instead (that's what's described in the issue).

Right. Whereas it would probably be better to have brew issue a warning pointing them to the full racket installation, even though that's a cask and not a formula.

tl;dr It feels like a homebrew problem, and one they're aware of.

**Comment by jryans** [comment]
The (confusing and incorrect) Homebrew deprecation warning [has been fixed](https://github.com/Homebrew/brew/pull/14450), so hopefully people are slightly more likely to get the right Racket distribution there in the future.

**Comment by nvlled** [comment]
I also had no idea how to install scribble since I wasn't familiar with the racket tooling, but just wanted to try out scribble.

After some fumbling around, I made installation instructions for ubuntu/debian systems: https://gist.github.com/nvlled/f9898acd0ef18d45406b5669ec44e3a0
